root/layout/index.js

```diff
@@ -91,9 +102,12 @@ const Layout = ({$c, ...props}) => (
     <body>
       <Header {...props} />
 
-      {!getRequestCookie($c.req, 'server_details_dismissed_mtime') && <ServerDetailsBanner />}
+      {!getRequestCookie($c.req, 'server_details_dismissed_mtime') &&
+        <ServerDetailsBanner />}
 
-      {!!($c.stash.alert && $c.stash.alert_mtime > getRequestCookie($c.req, 'alert_dismissed_mtime', 0)) &&
+      {!!($c.stash.alert && $c.stash.alert_mtime > getRequestCookie(
+        $c.req, 'alert_dismissed_mtime', 0,
```

Comment:
Maybe break after the `>` above and do
```suggestion
          getRequestCookie($c.req, 'alert_dismissed_mtime', 0)) &&
```